appcues/src/main/java/com/appcues/ui/extensions/StyleComponentExt.kt

```diff
@@ -164,12 +164,12 @@ internal fun ComponentStyle.getVerticalAlignment(default: Alignment.Vertical): A
     }
 }
 
-internal fun ComponentStyle.getHorizontalAlignment(): Alignment.Horizontal {
+internal fun ComponentStyle.getHorizontalAlignment(default: Alignment.Horizontal): Alignment.Horizontal {
```

Comment:
changed this because I wanted to be able to pass in `Start` as the default, rather than `CenterHorizontally`, for the label components (which also matches iOS default)